packages/flame/lib/src/components/parallax_component.dart

```diff
@@ -32,13 +31,14 @@ extension ParallaxComponentExtension on Game {
       images: images,
     );
 
-    return component..size.setFrom(componentSize);
+    return component;
   }
 }
 
 /// A full parallax, several layers of images drawn out on the screen and each
 /// layer moves with different velocities to give an effect of depth.
 class ParallaxComponent extends PositionComponent {
+  bool isFullscreen = true;
```

Comment:
shouldn't this be a private field?

Reply:
Yeah, maybe there is no usecase for changing the fullscreen mode after initialization.

Reply:
could be a getter function calculated over size!=null